ql/src/test/queries/clientpositive/like_control_characters.q

```diff
@@ -0,0 +1,15 @@
+set hive.mapred.mode=nonstrict;
+set hive.explain.user=false;
+set hive.vectorized.execution.enabled=true;
+
+create temporary table foo (col string) stored as orc;
+create temporary table bar (col binary) stored as orc;
+
+-- SORT_QUERY_RESULTS
+
+INSERT INTO bar select unhex('6162636465-166676869');
```

Comment:
Maybe just a question: can we use other way instead of the udf `unhex` to insert the control characters?
I want to keep the qtest more lightweight and independent.

Reply:
I thought so, but I couldn't think of any other way to insert control characters. Do you have any ideas?

Reply:
I changed it to use `LOAD DATA`.